index.html

```diff
@@ -10,15 +10,12 @@
       rel="stylesheet">
 </head>
 <body>
-  <header>
-    <h2>John Doe</h2>
-    <!-- <i class="material-icons">menu</i> -->
+  <nav class="flex">
+    <h2 class="flex"><a href="#" target="_blank">John Doe</a></h2>
     <img class="hover-button" src="/Icon-Menu.png" alt="">
```

Comment:
- [ ] The hamburger icon still not showing in your project. I think the reason for this is `/`. You can take a hint ( you can remove `/` or use `./` instead of `/`).

Reply:
Ok, I already did.